Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get update \
   && apt-get install --no-install-recommends -y \
   postgresql-client-17 \
   p7zip-full \
+  dcm2niix \
```

Comment:
This is most likely a very old version (as most stuff from apt). But it is also distributed over PyPI https://pypi.org/project/dcm2niix/ Use that one.